src/system/SystemConfig.h

```diff
@@ -733,7 +733,7 @@ struct LwIPEvent;
  *  Defaults to enabled on Zephyr platforms that do not enable Zephyr POSIX layer.
  */
 #ifndef CHIP_SYSTEM_CONFIG_USE_ZEPHYR_SOCKETS
-#if CHIP_SYSTEM_CONFIG_USE_SOCKETS && defined(__ZEPHYR__) && CONFIG_NET_SOCKETS_POSIX_NAMES
+#if CHIP_SYSTEM_CONFIG_USE_SOCKETS && defined(__ZEPHYR__) && defined(CONFIG_NET_SOCKETS_POSIX_NAMES)
```

Comment:
Again, is the contract that prj.conf settings end up as defined-or-not, without boolean values?

Reply:
Boolean Kconfig macros are either defined as true or not defined at all.